src/file_input/MIDI_file_class.py

```diff
@@ -92,6 +96,28 @@ def calc_time_delay(self):
         if self.curr_tempo is None or self.curr_time_signature is None:
             return None
         
+    
+    def is_correct_hand(self, note_num):
+        """
+        NOTE: The hand_to_play side refers to the side the PIANO is going to play.
+        The player would, presumably, be playing the opposite.
+
+        
+        If the given note is not on the hand_to_play side return false
+        """
+        #print(f'Hand to play: {self.hand_to_play}')
+        if self.hand_to_play == "right":    # The piano will play right hand
+            if note_num >= 60:
```

Comment:
Can we make this a class variable so that it could be editable in the future?

Reply:
Yeah I can do that, but it does bring up a few things. Right now, when a song is played the hand can only be set when first parsed. I can change that, but I'm not sure if the song objects persist after they are stopped.